CMSIS/Core_A/Source/irq_ctrl_gic.c

```diff
@@ -130,6 +130,7 @@ __WEAK uint32_t IRQ_GetEnableState (IRQn_ID_t irqn) {
 /// Configure interrupt request mode.
 __WEAK int32_t IRQ_SetMode (IRQn_ID_t irqn, uint32_t mode) {
   uint32_t val;
+  uint32_t valnn;
```

Comment:
Instead of creating the local variable `valnn`, is it possible to reuse `val` as it is done when reading/masking other IRQ modes?

Reply:
fixed

Reply:
Created new https://github.com/ARM-software/CMSIS_5/pull/845